app/views/docs/_definitions.html.erb

```diff
@@ -0,0 +1,41 @@
+<h2 id="definitions">Definitions</h2>
+
+<% @swagger.definitions.each do |definition| %>
+<h3 id="<%= definition.unique_key %>"><%= definition.name %></h3>
+
+<% if definition.description %>
+<%= definition.html_description %>
+<% end %>
+
+<code>
+  <pre>
+    {
+    <% definition.properties.each do |property| %>
+      <% if property.comment %>
+        /* <%= property.comment %> */
+      <% end %>
+      "<%= property.name %>": <%= property.sample_value %><%= ',' unless property == definition.properties.last %><% if property.nullable %> // nullable<% end %>
```

Comment:
any way to get this logic moved out of views and into something else where it is easier to see / test?